Video.js

```diff
@@ -289,6 +289,7 @@ Video.propTypes = {
   progressBarMarginBottom: PropTypes.number,
   stateOverlay: PropTypes.oneOf(['ACTIVE', 'INACTIVE', 'HIDDEN']),
   stateMiddleCoreControls: PropTypes.oneOf(['ACTIVE', 'INACTIVE', 'HIDDEN']),
+  overlayAutoHideTimeout: PropTypes.string,
```

Comment:
You should pass it as `PropTypes.number` 

https://facebook.github.io/react-native/docs/native-modules-android
Argument Types:
`Boolean -> Bool
Integer -> Number
Double -> Number
Float -> Number
String -> String
Callback -> function
ReadableMap -> Object
ReadableArray -> Array`

Reply:
Issue is that with number I will not be able to disable autohide after it has been set. Can this be a problem?

Reply:
OK. Changing String to number seems to work.